ui/package.json

```diff
@@ -83,7 +83,7 @@
     "globals": "^15.9.0",
     "jsdom": "^25.0.1",
     "msw": "^2.6.0",
-    "orval": "^7.3.0",
+    "orval": "7.3.0",
```

Comment:
A better fix would be to fix orval.config.ts to include output, instead of downgrading the dependency.

Reply:
@oshcherbakovv fyi